package.json

```diff
@@ -7,8 +7,7 @@
     "dist/js/subtitles*"
   ],
   "scripts": {
-    "test": "echo \"Error: no test specified\" && exit 1",
-    "prepare": "./run-docker-build.sh"
+    "test": "echo \"Error: no test specified\" && exit 1"
```

Comment:
> Also adds a prepare script to automatically build the library (reproducibly, via Docker) on npm pack, npm publish, and local no-arg npm install.

@rcombs It seems that the change has been undone in this commit

Reply:
Whoops! Fixed.